client/my-sites/plans/current-plan/style.scss

```
@@ -148,3 +146,136 @@
 	margin-bottom: 10px;
 	text-align: center;
 }
+
+body.is-section-plans.is-trial-plan {
+	&.theme-default.color-scheme {
```

This `.theme-default` selector feels like it might break under different Calypso colour schemes. I think we should remove it.
```suggestion
	&.color-scheme {
```